src/AutoRest.CSharp/Common/AutoRest/Plugins/NewProjectScaffolding.cs

```diff
@@ -363,7 +363,7 @@ private string GetBrandedTestCSProj()
             var writer = new CSProjWriter()
             {
                 TargetFrameworks = "$(RequiredTargetFrameworks)",
-                NoWarn = new("$(NoWarn);CS1591", "We don't care about XML doc comments on test types and members")
+                NoWarn = new("$(NoWarn);CS1591;Nu1605", "We don't care about XML doc comments on test types and members. We also don't care about System.ClientModel downgrade in the test project.")
```

Comment:
This should definitely not be here when we merge.  I am working on upgrading the System.ClientModel to 1.1.0-* https://github.com/Azure/autorest.csharp/issues/4242

Reply:
Pending on `System.ClientModel 1.1.0-*` release